stand/common/commands.c

```diff
@@ -238,7 +238,7 @@ command_commandlist(int argc __unused, char *argv[] __unused)
 		if (res)
 			break;
 		if ((*cmdp)->c_name != NULL && (*cmdp)->c_desc != NULL) {
-			snprintf(name, sizeof(name), "  %-15s  ",
+			snprintf(name, sizeof(name), "  %-20s",
```

Comment:
Previously the buffer was sized to exactly the right size (2 + 15 + 2 + 1). Now this is printing 2 + 20 + 1 = 23 characters, but the buffer is 24 characters long. Also, why is it correct to remove the trailing two spaces? You definitely want at least one to separate the name from the description. I imagine what you really want to do is put the spaces in their own pager_output calls so they don't get swallowed when a name is truncated.

Reply:
Sorry for the late response. It would definitely be better to put the trailing spaces in their own pager call, I didn't think of that.

As for why I removed the trailing spaces to begin with, it's because, as you said, they get truncated away anyways if the command is too long.

Oh, and I completely missed that buffer size thing.

Thanks!